HOW_TO_RELEASE.md

```diff
@@ -13,109 +13,133 @@ upstream        https://github.com/pydata/xarray (push)
-    release headline should get feedback from the team on what's important to include.
- 6. After merging, again ensure your main branch is synced to upstream:
-     ```sh
-     git pull upstream main
-     ```
- 7. If you have any doubts, run the full test suite one final time!
-      ```sh
-      pytest
-      ```
- 8. Check that the [ReadTheDocs build](https://readthedocs.org/projects/xray/) is passing on the `latest` build version (which is built from the `main` branch).
- 9. Issue the release on GitHub. Click on "Draft a new release" at
-    <https://github.com/pydata/xarray/releases>. Type in the version number (with a "v")
-    and paste the release summary in the notes.
- 10. This should automatically trigger an upload of the new build to PyPI via GitHub Actions.
-    Check this has run [here](https://github.com/pydata/xarray/actions/workflows/pypi-release.yaml),
-    and that the version number you expect is displayed [on PyPI](https://pypi.org/project/xarray/)
-11. Add a section for the next release {YYYY.MM.X+1} to doc/whats-new.rst (we avoid doing this earlier so that it doesn't show up in the RTD build):
-     ```rst
-     .. _whats-new.YYYY.MM.X+1:
-
-     vYYYY.MM.X+1 (unreleased)
-     -----------------------
-
-     New Features
-     ~~~~~~~~~~~~
-
-
-     Breaking changes
-     ~~~~~~~~~~~~~~~~
-
-
-     Deprecations
-     ~~~~~~~~~~~~
-
-
-     Bug fixes
-     ~~~~~~~~~
-
-
-     Documentation
-     ~~~~~~~~~~~~~
-
-
-     Internal Changes
-     ~~~~~~~~~~~~~~~~
-
-     ```
-12. Commit your changes and push to main again:
-      ```sh
-      git commit -am 'New whatsnew section'
-      git push upstream main
-      ```
-    You're done pushing to main!
-
-13. Update the version available on pyodide:
-    - Open the PyPI page for [Xarray downloads](https://pypi.org/project/xarray/#files)
-    - Edit [`pyodide/packages/xarray/meta.yaml`](https://github.com/pyodide/pyodide/blob/main/packages/xarray/meta.yaml) to update the
-        - version number
-        - link to the wheel (under "Built Distribution" on the PyPI page)
-        - SHA256 hash (Click "Show Hashes" next to the link to the wheel)
-    - Open a pull request to pyodide
-
-14. Issue the release announcement to mailing lists & Twitter. For bug fix releases, I
-    usually only email xarray@googlegroups.com. For major/feature releases, I will email a broader
-    list (no more than once every 3-6 months):
-      - pydata@googlegroups.com
-      - xarray@googlegroups.com
-      - numpy-discussion@scipy.org
-      - scipy-user@scipy.org
-      - pyaos@lists.johnny-lin.com
-
-    Google search will turn up examples of prior release announcements (look for
-    "ANN xarray").
-    Some of these groups require you to be subscribed in order to email them.
+1. Ensure your main branch is synced to upstream:
+
+   ```sh
+   git switch main
+   git pull upstream main
+   ```
+
+1. Add a list of contributors.
```

Comment:
Forgotten the enumeration again.

Reply:
This is rendering fine. It's a trick to have the renderer do the numbering without having to do it manually

Reply:
the downside of that trick is that reading the source becomes a bit harder to read (which item from the top was that again?), so I'd prefer a linter / autoformatter to keep source and rendered version as close as possible.

Given that [`prettier`](https://github.com/prettier/prettier) ([mirror](https://github.com/rbubley/mirrors-prettier)) appears to get this right, maybe we should be using that instead? That would also auto-format yaml, css and json, and for toml we can try [taplo](https://github.com/tamasfe/taplo/) ([mirror](https://github.com/ComPWA/taplo-pre-commit)).